low-level/framework/utils/service_logging.py

```diff
@@ -19,130 +19,33 @@
  ****************************************************************************
 """
 
-import logging.handlers
-import time
+import sys
 import os
+import syslog
+from cortx.utils.log import Log
+
+
+def init_logging(service_name, file_path, log_level="INFO"):
+    """Initialize logging for SSPL component."""
+    # Log rotation is configured within cortx-utils with
```

Comment:
> **cdeshmukh**: what about log rotation config? What are the defaults, if any exists, does it server SSPL needs?
> 
> **gargpalak048:** Log rotation is handled by cortx-utils itself.
> Default config in cortx-utils is,
> backup_count=10,
> file_size_in_mb=10,
> max_bytes = file_size_in_mb * 1024 * 1024
> most of the components like HA using this default config only, I think for SSPL also this will suffice
> 
> **cdeshmukh**: yes that suffice. but still add comments about default config around this code and also mention its recommended to refer to latest code for confirming config. that may help in future

Done